flink-scala-shell/src/main/scala/org/apache/flink/api/scala/FlinkShell.scala

```diff
@@ -255,14 +257,25 @@ object FlinkShell {
     yarnConfig.queue.foreach((queue) => args ++= Seq("-yqu", queue.toString))
     yarnConfig.slots.foreach((slots) => args ++= Seq("-ys", slots.toString))
 
+    val customCommandLines = CliFrontend.loadCustomCommandLines(
+      configuration,configurationDirectory)
+    val commandOptions = CliFrontendParser.getRunCommandOptions
+    val customCommandLineOptions = new Options()
+    customCommandLines.asScala.foreach(cmd => {
```

Comment:
this is already done int he `CliFrontend` constructor. It may be sufficient to switch the initialization of `frontend` and `commandLine`.

Reply:
@zentol  I have fixed that, but I have to make `customCommandLineOptions` of `CliFrontend` to be static (a little ugly)